source/guides/aggregate/adding-cosginatures-aggregate-complete.rst

```diff
@@ -0,0 +1,92 @@
+:orphan:
+
+.. post:: 13 July, 2020
+    :category: Aggregate Transaction
+    :tags: SDK, CLI
+    :excerpt: 1
+    :nocomments:
+
+#############################################################
+Adding cosignatures to aggregate complete trasactions offline
+#############################################################
+
+This guide will show you how to add cosignatures to an aggregate complete transaction while offline, then use an online device to announce the transaction.
```

Comment:
This guide will show you how to add cosignatures to an aggregate complete transaction without using the partial caché.